config/confd/templates/cloudfront-pk.pem.tmpl

```diff
@@ -0,0 +1 @@
+{{if exists "/resin/registry2/s3/cloudfront/privatekey"}}{{getv "/resin/registry2/s3/cloudfront/privatekey"}}{{end}}
```

Comment:
I think checking for existence of a key where the parent also does not exist, fails with an error. That is, in the check above, if `/resin/registry2/s3/cloudfront` doesn't exist, this will error out instead of evaluating to `false`. I'd need to double-check though to confirm.